tests/test_importers.py

```diff
@@ -115,11 +127,12 @@ def _class(manager):
 
 
 class TestImporters(TestBaseImporters):
+    CARD_IMPORTERS = ['encryptr',]
 
     def test_importers(self):
         """Testing: importer parse method using real data."""
         keys = ['title', 'password', 'login', 'ssid']
-        ignore = ['networkmanager']
+        ignore = ['networkmanager', 'card']
```

Comment:
There is no `card` importer in `pass_import.importers`, so no need to ignore it here.